profiling.qmd

```diff
@@ -38,43 +37,121 @@ import ProgressMeter
+
+```{julia}
+show(unique(pr01.tbl.p))
+```
+
+To reiterate, the first row contains the parameter estimates for this model fit to the original response values with the constraint that $\sigma=0.130088$, instead of the global estimate $\hat{\sigma}=0.139458$ in the row for which $\zeta=0.0$.
+
+The global estimates are included in every block at the row for which $\zeta=0.0$.
+
+```{julia}
+filter(r -> iszero(r.ζ), pr01.tbl)
+```
+
+The $\zeta$ column in this table is a measure of the quality of the fit from the parameters in each row, relative to the global parameter estimates, as measured by the change in the objective (negative twice the log-likelihood).
+
+The minimum value for the objective is that at the global parameter estimates.
+The change in the objective when we constrain one parameter to a particular value has approximately a $\chi^2$ distribution on 1 degree of freedom, which is the square of a standard normal distribution, $\mathcal{Z}^2$.
+We can convert this change in the quality of the fit to the scale of the standard normal distribution by taking the *signed square root*, which is the square root of the change in the objective with the sign of $\psi-\hat{\psi}$ where $\psi$ represents the parameter being profiled.
+This is the value labelled $\zeta$ in the table.
+
+To review:
+
+- Each row in the table is the result of re-fitting the original model with the parameter in the `p` column held fixed at a particular value, as shown in the column for that parameter.
+
+- The $\zeta$ column is the signed square root of the change in the objective from the global parameter estimates.
+
+- Thus in the block of rows where $\sigma$ is held fixed, the $\zeta$ values in rows for which $\sigma<\hat\sigma$ are negative and those for which $\sigma > \hat\sigma$ have positive values of $\zeta$.
+
+- Rows in which $\zeta=0.0$ are the global parameter estimates.
+
+## Profile zeta plots
+
+@fig-kb07zetabeta shows, for each of the fixed effects parameters, $\zeta$ versus the parameter value.
 
 ```{julia}
-confint(pr01)  # defaults to the 95% confidence level
+#| code-fold: true
+#| fig-cap: "ζ versus the value of the coefficient for the fixed-effects parameters in a model of response speed for the kb07 data."
+#| label: fig-kb07zetabeta
+zetaplot!(Figure(; resolution=(1200, 350)), pr01; ptyp='β')
 ```
 
-We can assess the symmetry and range of these intervals in a plot of the square root of the change in negative twice the log-likelihood from the global parameter estimate (again, details are given below).
-On the signed square root scale this quantity is called $\zeta$ because it is comparable to a standard normal deviate, which is usually written as `z`.
-@fig-kb07abszetabeta shows the absolute value of $\zeta$ for each of the fixed-effects coefficients in the model.
+The lines on these panels are read like normal probability plots.
```

Comment:
for statistical novices, it might be good to say "QQ plot" directly

Reply:
Will do.